### PR TITLE
Task list

### PR DESCRIPTION
**Pull request by NikitaSedyx**

### CODE REVIEW

File: app/components/groups/group.contoller.js [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+
+    .controller("GroupController", GroupController)
+
+    GroupController.$inject = ["$scope"]
+
+    function GroupController($scope){
+      $scope.groupView = "list"
+      $scope.groups = [{
+          title: "First",
+          tasks: [{description: "firstklrgsdgsdgfkdgfkldgfkdgf sadasdsasddas asdasdasd godgfpodgf po asdp[odasokdfigjdfisuj dfsihuodf shjuodhfisjl] description"}],
```
Comment (ghost) [inline app/components/groups/group.contoller.js]: Don't leave any data in controller. create a service, where you make calls to server for data and you can put there dummy data also. BTW, you can use $httpBackend for unit tests and mock this call while no server code is ready

Comment (NikitaSedyx) [inline app/components/groups/group.contoller.js]: it's only for test...

File: index.jade [hunk 2]
```diff
@@ -31,6 +31,7 @@ html(ng-app="todo")
           div(ui-view)
 
     script(src="assets/libs/jquery/dist/jquery.min.js")
+    script(src="assets/libs/masonry/dist/masonry.pkgd.js")
```
Comment (ghost) [inline index.jade:34]: masonry? it's for ui layout?

Comment (NikitaSedyx) [inline index.jade:34]: yes, for group-panel view (the second image)

File: assets/sass/_groups.sass [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+  .text
+    text-indent: 5px
+
+  .date
+    font-size: 14px
+    font-style: italic
+
+  .date:before
+    content: "Updated at: "
```
Comment (ghost) [inline assets/sass/_groups.sass:29]: what if I need to display date without such label? maybe it's a part of jade layout? or specific `.updated-date` class case

Comment (NikitaSedyx) [inline assets/sass/_groups.sass:29]: it's works only when selector is .group .date, but updated-date describes better, I'll fix it

**Pull request merged**